src/main/java/com/taskadapter/redmineapi/internal/Transport.java

```diff
@@ -602,6 +606,19 @@ public void setPassword(String password) {
 	public void setLogin(String login) {
 		setCredentials(login, password);
 	}
+        /**
+         * @return the impersonateUser
+         */
+        public String getImpersonateUser() {
+           return impersonateUser;
+        }
+
+        /**
+        * @param impersonateUser the impersonateUser to set
+        */
```

Comment:
On Sun, 2015-01-11 at 11:35 -0800, Alexey wrote:
> In src/main/java/com/taskadapter/redmineapi/internal/Transport.java:
> 
> > @@ -602,6 +606,19 @@ public void setPassword(String password) {
> >  	public void setLogin(String login) {
> >  		setCredentials(login, password);
> >  	}
> > +        /**
> > +         * @return the impersonateUser
> > +         */
> > +        public String getImpersonateUser() {
> > +           return impersonateUser;
> > +        }
> > +
> > +        /**
> > +        * @param impersonateUser the impersonateUser to set
> > +        */
> 
> this javadoc does not explain anything. what is this user for?
> use-cases? how to use it?

Hi,

I would like to avoid duplicating the redmine documentation within the
javadoc. Do you think a link to the User-Impersonation Feature of
redmine would be sufficient?
> 
http://www.redmine.org/projects/redmine/wiki/Rest_api#User-Impersonation
http://www.redmine.org/issues/11755